main.go

```diff
@@ -100,6 +107,12 @@ func main() {
 	healthModule{}.DeprecatedRegister(r)
 	healthModule{}.Register(r.Group("/api"))
 
+	if config.HTTP.OIDC.Enable {
+		rsakeys := GetOidcPublicKeys(config.HTTP.OIDC)
+		r.Use(VerifyTokenMiddleware(config.HTTP.OIDC,rsakeys))
+		SubscribeToKeyURLUpdates(config.HTTP.OIDC,rsakeys)
```

Comment:
Suggest running `goimports -w .` to catch all non-conventional formatting like the lack of spaces here.

Reply:
Where does the goimports exe come from?

Reply:
Running `go get golang.org/x/tools/cmd/goimports` in terminal/cmd should be enough to install it I believe.

Reply:
Do we have this set as one of the rules for linting that the whole team always uses? If so where is that stated? Can we add it to the readme? Did we even have a discussion about this?
Most of the project appears to be in violation to this paradigm. Will not implement here.

Reply:
Have mentioned it from time to time. We do not have it documented, but you're free to add it to a CONTRIBUTING.md and/or https://github.com/iver-wharf/iver-wharf.github.io repo

We do have a list of rules that we try to follow, which was declared long ago, before we even moved to GitHub. Have an issue on it here https://github.com/iver-wharf/iver-wharf.github.io/issues/69

Reply:
In https://github.com/iver-wharf/wharf-api/pull/106 I've added a GitHub action to ensure this using the same goimports util